AgentDiscoveries-Frontend/.gitignore

```diff
@@ -3,3 +3,9 @@ target/
 node/
 node_modules/
 npm-debug.log
+
+app/src/components/utilities/request-helper.js
+package-lock.json
+package.json
+pom.xml
+webpack.config.js
```

Comment:
All of these files should be kept: config files in general should be kept. That includes:

* package-lock.json
* package.json
* pom.xml
* webpack.config.js

Also, your request-helper.js is just straight up code so that should end up in the repository too!